editor/js/Loader.js

```diff
@@ -154,18 +154,7 @@ var Loader = function ( editor ) {
 
 					var contents = event.target.result;
 
-					var loader;
-
-					if ( isGLTF1( contents ) ) {
```

Comment:
It might be worth keeping the `isGLTF1` function and displaying an alert if the user tries a glTF 1.0 file. We can suggest upgrading the file to glTF 2.0 using glTF-Pipeline, for example.

Reply:
Good idea 👍